app/helpers/application_helper.rb

```diff
@@ -265,4 +265,13 @@ def paginate(objects, *)
     result << " #{objects.total_count} records" if objects.total_pages > 1
     result
   end
+
+  def list_with_show_more(items, display_limit, show_more_tag = nil, ul_options = {}, &block)
+    li_tags = items.first(display_limit).map { |i| capture(i, &block) }
+    li_tags << show_more_tag if items.size > display_limit
```

Comment:
that `= nil` in the arguments would be weird here ... do we need to allow it to be nil ?